developers/weaviate/concepts/vector-index.md

```diff
@@ -28,9 +30,11 @@ Weaviate's vector-first storage system takes care of all storage operations with
 
 ## Why index data as vectors?
 
-Vectors are a great way to represent data objects' meaning, such as meaning of text or contents of images or video. They also enable determination of [semantic similarity](https://en.wikipedia.org/wiki/Semantic_similarity) between data objects.
+Vectors are a great way to represent data objects' meaning, such as meaning of text or contents of images or video. They also enable determination of [semantic similarity](https://en.wikipedia.org/wiki/Semantic_similarity) between data objects, which enables vector databases to retrieve data objects based on their similarity to a query vector - also called similarity search.
+
+To make this concept more tangible, think of vectors as coordinates in a *n*-dimensional vector space. For example, we can represent *words* in a 2-dimensional space. If you use an algorithm that learned the relations of words or co-occurrence statistics between words from a corpus (like [GloVe](https://github.com/stanfordnlp/GloVe)), then single words can be given the coordinates (vectors or [vector embeddings](https://weaviate.io/blog/vector-embeddings-explained#what-exactly-are-vector-embeddings)) according to their similarity to other words. These algorithms are powered by Machine Learning and Natural Language Processing concepts. 
 
-To make this concept more tangible, think of vectors as coordinates in a *n*-dimensional space. For example, we can represent *words* in a 2-dimensional space. If you use an algorithm that learned the relations of words or co-occurrence statistics between words from a corpus (like [GloVe](https://github.com/stanfordnlp/GloVe)), then single words can be given the coordinates (vectors) according to their similarity to other words. These algorithms are powered by Machine Learning and Natural Language Processing concepts. In the picture below, you see how this concept looks (simplified). The words `Apple` and `Banana` are close to each other. The distance between those words, given by the distance between the vectors, is small. But these two fruits are further away from the words `Newspaper` and `Magazine`.
+In the picture below, you see how this concept looks (simplified). The words `Apple` and `Banana` are close to each other. The distance between those words, given by the distance between the vectors, is small. But these two fruits are further away from the words `Newspaper` and `Magazine` in the same vector space.
```

Comment:
Consider:

In the graph below, the words `Apple` and `Banana` are shown close to each other. `Newspaper` and `Magazine` are also close to each other, but they are far away from `Apple` and `Banana`. 

Within each pair, the distance between words is small because the objects have similar vector representations. The distance between the pairs is larger because the difference between the vectors is larger. Intuitively, fruits are similar to each other, but fruits are not similar to reading material. 

For more details of this representation, see: [GloVe corpus](https://github.com/stanfordnlp/GloVe)) and [vector embeddings](https://weaviate.io/blog/vector-embeddings-explained#what-exactly-are-vector-embeddings.

Reply:
included